src/components/providers/ProviderCategoryWrapper.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+  import ServiceProviderCard from '@/components/cards/ServiceProviderCard.vue';
+  import { type ServiceProvider } from '@/stores/ServiceProviderStore';
+
+  defineProps<{
+    categoryTitle: string;
+    serviceProviders: Array<ServiceProvider>;
+  }>();
+</script>
+
+<template>
+  <v-container v-if="serviceProviders && serviceProviders.length > 0">
```

Comment:
```suggestion
  <v-container v-if="serviceProviders?.length > 0">
```
Reicht das in diesem Fall aus?

Reply:
Der ganze Container wird ausgeblendet, wenn es keine Service Provider gibt. Deshalb ist die Schulportaladministration wieder verschwunden.

Reply:
Tricky hier leider, weil der Title muss auch weg, wenn wir keine Provider haben. Allerdings muss der Title bleiben für die Kategorie "Verwaltung", da wir immer ein Provider haben... Ich habe deswegen für diese Kategorie kein Wrapper benutzt.

Reply:
Alternativ kann man den` <slot>` 2 mal in der Wrapper-komponente definieren, im ersten Fall, wenn wir Provider haben und im zweiten fall, wenn wir keine Provider haben aber hier dann wieder den title dafür definieren. Ich ging aber mit der ersten Lösung da "sauberer".